build.gradle.kts

```diff
@@ -208,9 +209,13 @@ tasks.register("releaseToSdkMan") {
     //   - 1.3.RC5: not final
     //   - 1.3-alpha5: not final
     val isFinalRelease = Regex("""[0-9\.]*""").matchEntire(versionString) != null
-    dependsOn(tasks.withType<SdkReleaseVersionTask>())
-    if (isFinalRelease) {
-        dependsOn(tasks.withType<SdkDefaultVersionTask>())
-        dependsOn(tasks.withType<SdkAnnounceVersionTask>())
+    // We don't publish snapshots and alphas at all to SDKman.
```

Comment:
LGTM, I think we also need to change the comment, esp. the `1.3-alpha5`.

Reply:
How about now?